website/public/js/controllers/challengesCtrl.js

```diff
@@ -277,6 +279,28 @@ habitrpg.controller("ChallengesCtrl", ['$rootScope','$scope', 'Shared', 'User',
       $scope.search.group = _.transform($scope.groups, function(m,g){m[g._id] = false});
     }
 
+    $scope.shouldShow = function(task, list, prefs){
+      return true;
+    };
+
+    //A helper function to dermine if the user has enough gems to create a Habitrpg challenge
+    $scope.hasEnoughGems = function(gid) {
```

Comment:
That makes sense. I'm unsure of which is better. I could see use for this on both sides. Is there a benefit to taking the function off scope?